### ISSUE TITLE
[Components and pattern standards] Navigation mechanisms aren't consistent across pages. (04.10.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#56069]
#### VFS team name
Benefits Team 1

#### VFS product name
Supplemental Claims Form 20-0995

#### VFS feature name
Supplemental Claims Form 20-0995

#### Point of Contact/Reviewers
Sara Sterkenburg - @sterkenburgsara - IA

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Navigation mechanisms aren't consistent across pages.

### Issue Details
The introduction page title to this form reads:  "Is this the form I need?" which does not match breadcrumb or URL for this page. This naming misalignment in primary navigation  can cause user confusion, because it doesn't clearly confirm to users that they are in the right place.

### Link, screenshot or steps to recreate
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/43919-Sara-041001.png

### VA.gov Experience Standard
[Category Number 04, Issue Number 10](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.3_AA

### Platform Recommendation
H1 of the page must match other navigational elements and should read "File a Supplemental Claim" or something very similar. If you need to ask the question "Is this the form I need?" on the page, it would be better to stub out a section beneath the primary page header and make this an H2. Content folks on sitewide CAIA should be able to support modifications there as needed. In summary, the heading structure for this intro page should read: 
* H1 File a Supplemental Claim
* H2 Is this the form I need?
* H2 What type of claim are you filing a Supplemental Claim for?

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by eileen-coforma** [comment]
This has already been fixed: https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/start
